client/client.go

```diff
@@ -18,6 +18,7 @@ type Client struct {
 }
 
 // New returns a new Miniflux client.
+//
```

Comment:
This additional row makes lsp hints show the deprecated notice
![image](https://github.com/user-attachments/assets/047d229e-90cc-4875-bca4-a845b16d6e61)